rt-forecast-2/format/utils/format_forecast_us.R

```diff
@@ -0,0 +1,68 @@
+
+# Packages ----------------------------------------------------------------
+require(EpiNow2)
+require(data.table)
+require(lubridate)
+source(here::here("utils/dates-to-epiweek.R"))
+source(here::here("utils/get-us-data.R"))
+
+# forecast_date -> date forecast was made
+format_forecast_us <- function(forecasts, 
+                               forecast_date = NULL, submission_date = NULL){
+  
+  # Filter to full epiweeks
+  forecasts1 <- dates_to_epiweek(forecasts)
```

Comment:
with data table it can be. a good idea to explicitly copy (data.table::copy). I would probably just stick to the forecast variable throughout this function without adding tmp names.

Reply:
oops! i  meant to remove - was actually just using the temp names while testing it. my bad!